src/Illuminate/Queue/DatabaseQueue.php

```diff
@@ -187,15 +187,24 @@ protected function releaseJobsThatHaveBeenReservedTooLong($queue)
     {
         $expired = Carbon::now()->subSeconds($this->expire)->getTimestamp();
 
-        $this->database->table($this->table)
-                    ->where('queue', $this->getQueue($queue))
+        $expiredJobs = $this->database->table($this->table)
+                    ->where('queue', $queue)
```

Comment:
I'm not quite sure about this change to fix the double-calling of `getQueue()`.
It makes sense to just use the `$queue` parameter as the method is getting the proper value. It's protected so it won't be called from outside.
But there are other calls to `getQueue()` in different places. I think the public methods should be calling `getQueue()` but the protected (or private) should rely on the proper value being passed to them. This would make the usage of `$query` more consistent.